gbif-occurrence-linkage/main.py

```diff
@@ -2,49 +2,60 @@
 import logging
 import os
 import uuid
+
+import pika
+import requests as requests
 from typing import Dict
+from pika.amqp_object import Method, Properties
+from pika.adapters.blocking_connection import BlockingChannel
 
-import requests
-from kafka import KafkaConsumer, KafkaProducer
 import shared
 
 logging.basicConfig(format="%(asctime)s - %(message)s", level=logging.INFO)
 
 
-def start_kafka() -> None:
+def run_rabbitmq() -> None:
     """
-    Start a kafka listener and process the messages by unpacking the image.
-    When done it will republish the object, so it can be validated and storage by the processing service
+    Start a RabbitMQ consumer and process the messages by unpacking the image.
+    When done, it will publish an annotation to annotation processing service
     """
-    consumer = KafkaConsumer(
-        os.environ.get("KAFKA_CONSUMER_TOPIC"),
-        group_id=os.environ.get("KAFKA_CONSUMER_GROUP"),
-        bootstrap_servers=[os.environ.get("KAFKA_CONSUMER_HOST")],
-        value_deserializer=lambda m: json.loads(m.decode("utf-8")),
-        enable_auto_commit=True,
-    )
-    producer = KafkaProducer(
-        bootstrap_servers=[os.environ.get("KAFKA_PRODUCER_HOST")],
-        value_serializer=lambda m: json.dumps(m).encode("utf-8"),
+    connection = pika.BlockingConnection(
+        pika.ConnectionParameters(
+            os.environ.get("RABBITMQ_HOST"),
+            credentials=pika.PlainCredentials(
+                os.environ.get("RABBITMQ_USER"),
```

Comment:
What credentials are these? the same ones to log into the admin portal? down the line, can we make this a less privileged account?